test-suite/js/src/test/require-modules/org/scalajs/testsuite/jsinterop/ModulesTest.scala

```diff
@@ -0,0 +1,113 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2016, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.jsinterop
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation._
+
+import org.junit.Assert._
+import org.junit.Test
+
+class ModulesTest {
```

Comment:
Can you please add a comment stating that this relies on Node.js modules, because we currently have no reasonable way of loading non-built in modules and that it should be made VM independent once we have this.